modules/ROOT/pages/add-api-to-unified-schema.adoc

```diff
@@ -105,6 +106,16 @@ If you need to edit the URL of an API that you've added to the unified schema, y
 . Type in the new URL.
 . Click *Save*.
 
+[[change-the-authentication-method-for-an-api-aded-to-the-unified-schema]]
+== Change the Authentication Method for an API Added to the Unified Schema
+
+If you need to edit the authentication method of an API that you've added to the unified schema, you can do so on the *API details* page.
+
+. Click *List of APIs added* and select the appropriate API.
+. Click *API details*.
+. In the *Authentication* panel, select an authentication policy and complete any necessary fields.
```

Comment:
@IsaacEldridge, please, review if the functionality works as you described. As far as I've checked, if you have added an authentication method previously, when changing it, you should first remove the old configuration (Button "Remove and add a new one") and then add the new one. I'd turn the statement like this (or similar): "In the *Authentication* panel, remove the old authentication, select the new authentication policy and complete any necessary fields."